installer/functions

```diff
@@ -156,7 +193,9 @@ release() {
 # sure the kernel does not panic (this is the default configuration of a vanilla
 # kernel). See crbug.com/260955 for details.
 disablehungtask() {
-    echo 0 > /proc/sys/kernel/hung_task_panic
+    # Don't fail since we do not have permission to write /proc/sys/* if we are
+    # in container.
+    echo 0 2>/dev/null > /proc/sys/kernel/hung_task_panic || true
```

Comment:
I did this, but it doesn't work on my machine. It still printed out the error messages, and I'm not sure if it's the same on ChromeBook.
I guess what it did is something like this

    open("proc/sys/kernel/hung_task_panic")
    dup2()
    open("/dev/null")
    dup2()

and the error occurs (at first `open`) before we redirect the stderr.